chain-tx-enclave-next/tx-validation-next/src/sgx_module.rs

```diff
@@ -19,6 +19,7 @@ use std::sync::Arc;
 
 /// FIXME: genesis app hash etc.?
 pub const NETWORK_HEX_ID: u8 = get_network_id!();
+pub const TDBE_MRENCLAVE: &str = env! {"TDBE_MRENCLAVE"};
```

Comment:
maybe similar to https://github.com/crypto-com/chain/issues/2174 ?
```suggestion
pub const TDBE_MRENCLAVE: [u8; 32] = include_bytes! {"tdbe.mrenclave"};
```